app/controllers/items_controller.rb

```diff
@@ -39,6 +39,6 @@ def destroy
   private
 
   def item_params
-    params.require(:item).permit(:user)
+    params.require(:item).permit(:name, :user)
```

Comment:
`:user` can be removed (keep `:name`) as there is no user field submitted in the item form.